iset.mm

```diff
@@ -99786,6 +99786,18 @@ seq K ( .+ , G , S ) ) $=
       TJOKMASTPQR $.
   $}
 
+  ${
+    $d F x y $.  $d M x y $.  $d ph x y $.
+    iseqseq3.m $e |- ( ph -> M e. ZZ ) $.
+    iseqseq3.f $e |- ( ( ph /\ x e. ( ZZ>= ` M ) ) -> ( F ` x ) e. CC ) $.
+    $( Equality using four argument or three argument sequence builders.
```

Comment:
@avekens : this is `seq M ( +. , F , S )` versus `seq M ( +. , F )`
@jkingdon : maybe write: "four-argument or three-argument sequence builders ( ~ df-... or ~ df-... )."

Reply:
I'm usually a fan of having the comment restate or explain the symbols, rather than just repeat what is in the formal statement of the theorem. But this is a case where I'm leaning towards just saying

```
Equality of ` seq M ( + , F , CC ) ` and ` seq M ( + , F ) `
```

rather than getting into counting arguments or pointing to definitions.

The good news is that this whole distinction should go away once we switch everything (or almost everything) to ` seq M ( + , F ) ` .